pennylane_orquestra/ibmq_device.py

```diff
@@ -22,15 +22,36 @@
 
 
 class QeIBMQDevice(OrquestraDevice):
-    """The Orquestra IBMQ device."""
+    """The Orquestra IBMQ device.
+
+    Args:
+        wires (int, Iterable[Number, str]]): Number of subsystems represented
+            by the device, or iterable that contains unique labels for the
+            subsystems as numbers (i.e., ``[-1, 0, 2]``) or strings (``['ancilla',
+            'q1', 'q2']``). Default 1 if not specified.
+        shots (int): number of circuit evaluations/random samples used to estimate
+            expectation values of observables
+        backend (str): the name of the Qiskit backend to use supported by
+            Orquestra, e.g., ``"ibmq_qasm_simulator"`` or the name of real hardware
+            devices
+
+    Keyword Args:
+        ibmqx_token=None (str): the authentication token needed to run a job on
+            IBMQ which is submitted to Orquestra as part of the workflow
```

Comment:
```suggestion
            IBMQ
```
(the orquestra part of the sentence seems a bit superfluous?)